content/1.7/back-office/customers/customers/outstanding-listing.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Outstanding
+weight: 1
+---
+
+# SELL > CUSTOMERS > Outstanding
+
+![Outstanding page](static/img/customers-outstanding.png)
+
+This UI is shown after Enabling the _Enable B2B mode_ toggle switch in _CONFIGURE > Customer Settings_ and it allows to track the B2B customer behavior. 
+
+## Configuration of the new B2B input fields in Customer edit page
+
+Once the B2B mode is enabled, new input fields are displayed in the Customer edit page by navigating to _Customers > Customers_ and editing the desired Customer. Customer must be registered in Prestashop himself, of added by Administrator. New input fields are the following:
+- **Allowed outstanding amount** - input can be filled with valid characters: 0-9 only. Default value is 0. If characters are typed as latin letters, input erases the wrong text automatically. If there are more than 19 characters in the field, the instant error is displayed below the field - exclamation mark and message _This field is invalid._ 
+- **Maximum number of payment days** - limit for Customer purchases in days. Default value is 0. If characters are typed as latin letters, there will be a browser-type notification near the field, called _Please enter a number._ If there are more than 10 characters typed in this field, there will be a Prestashop error prompted.
```

Comment:
@MatShir there is an error, inputting more than 10 characters in this field. The outcome in Prestashop and Database:
<img width="1220" alt="image" src="https://user-images.githubusercontent.com/29095235/151391759-af7c77f5-7bce-4db6-9c14-0b8066bd59b0.png">

<img width="358" alt="image" src="https://user-images.githubusercontent.com/29095235/151392475-595054fb-c8aa-49c6-8031-1f0b014105e6.png">

Max lenght is added in Database, but not in Prestashop notifications:

<img width="439" alt="image" src="https://user-images.githubusercontent.com/29095235/151392732-4cc48d15-7c4f-4d1a-b9df-e34c2ae5fad2.png">